src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -94,10 +94,17 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             XContentParser parser = request.contentParser();
             ensureExpectedToken(XContentParser.Token.START_OBJECT, parser.nextToken(), parser);
             Template template = Template.parse(parser);
-            boolean dryRun = request.paramAsBoolean(DRY_RUN, false);
+            boolean validation = request.paramAsBoolean(VALIDATION, true);
```

Comment:
So after thinking on this for a while, I think I like "validation" as a name but would like it to be more fine-grained.  We don't need to do all the choices for 2.12, but can we change this to a `String` parameter defaulting to "all"?

That will leave the door open to adding more options in the future like maybe validating everything except plugins "-plugins" or only version compatibility, etc.

Reply:
For clarity the (future) capability I'm envisioning is similar to the OpenSearch [Nodes Stats API](https://opensearch.org/docs/latest/api-reference/nodes-apis/nodes-stats/) where you can specify a list of stats you want, or "all" to get all of 'em.